httptransport/client/matcher.go

```diff
@@ -147,13 +150,14 @@ func (c *HTTP) UpdateOperations(ctx context.Context, updaters ...string) (map[st
 }
 
 // LatestUpdateOperations returns the most recent UpdateOperation per updater.
-func (c *HTTP) LatestUpdateOperations(ctx context.Context) (map[string][]driver.UpdateOperation, error) {
+func (c *HTTP) LatestUpdateOperations(ctx context.Context, k driver.UpdateKind) (map[string][]driver.UpdateOperation, error) {
 	u, err := c.addr.Parse(httptransport.UpdateOperationAPIPath)
 	if err != nil {
 		return nil, err
 	}
 	v := url.Values{}
 	v.Add("latest", "true")
+	v.Add("kind", string(k))
```

Comment:
I think we talked about special casing an empty driver.UpdateKind ( value: "" ) to signal that the UpdateKind "vulnerability" should be used. 

https://github.com/quay/claircore/blob/main/internal/vulnstore/postgres/getupdateoperations.go

In these, and similar methods, I think we want to send off a "vulnerability" value of driver.UpdateKind as the "default", so that when it hits the ClairCore code, we maintain API compatibility. 

i.e. 

```
switch driver.UpdateKind {
case "" || driver.VulnerabilityKind:
    ... 
    v.Add("kind", "vulnerability")
case driver.EnrichmentKind:
   v.Add("kind", "enrichment")
default:
   return nil, fmt.Errorf("wtf")
}
```

Reply:
scrolled down a bit and saw you handle this in the actual handler, that's fine too. Maybe worth doing it in both places for self-documentation?